lib/Elevate/Leapp.pm

```diff
@@ -87,6 +103,70 @@ sub upgrade ($self) {
     return;
 }
 
+sub search_report_file_for_blockers ( $self, @ignored_blockers ) {
```

Comment:
Out of scope for this case, but we should refactor  `_report_leapp_failure_and_die()` to use this so that we provide info about all the inhibitors to the customer in the event that leapp upgrade fails.

@timmullin - Would you mind filing a follow up case to refactor `_report_leapp_failure_and_die()` to use `search_report_file_for_blockers()` please?

Reply:
Filed as RE-217